ReactWindows/ReactNative/Views/TextInput/ReactTextInputManager.cs

```diff
@@ -513,6 +513,17 @@ public void SetImportantForAccessibility(ReactTextBox view, string importantForA
             AccessibilityHelper.SetImportantForAccessibility(view, importantForAccessibility);
         }
 
+        /// <summary>
+        /// Controls the visibility of the DeleteButton.
+        /// </summary>
+        /// <param name="view">The view instance.</param>
+        /// <param name="clearButtonMode">Visibility of the DeleteButton.</param>
+        [ReactProp("clearButtonMode")]
+        public void SetClearButtonMode(ReactTextBox view, string clearButtonMode)
```

Comment:
add to WPF ReactTextInputManger as well, please.

Reply:
Added

Reply:
@matthargett I tested in WPF playground and the changes have no effect, which is expected because TextInputs in WPF don't seem to have the default clear buttons like they do in UWP. I can confirm that WPF runs smoothly with the changes, and passing the clearButtonMode prop will not break anything.